tests/cloudpickle_test.py

```diff
@@ -1008,7 +1008,8 @@ def example():
 
         # choose "subprocess" rather than "multiprocessing" because the latter
         # library uses fork to preserve the parent environment.
-        command = ("import pickle, base64; "
+        command = ("import base64; "
+                   "from cloudpickle.compat import pickle; "
```

Comment:
In both of these test cases, we are using the highest supported protocol with `cloudpickle`, which is protocol `5`, to produce the pickled data. So we wound up needing to change these since we are trying to load the pickled data and then need to use `pickle5` when `pickle` doesn't have protocol `5` support. We could alternatively skip the test, restrict the protocol based on `pickle.HIGHEST_PROTOCOL`, or something else. We could also leave as-is. Some options to consider 🙂

Reply:
In this case I agree. Thanks.